hex-arch-ports/src/commonMain/kotlin/dev/eskt/arch/hex/port/MultiStreamTypeEventListener.kt

```diff
@@ -4,6 +4,6 @@ import dev.eskt.store.api.EventEnvelope
 import dev.eskt.store.api.StreamType
 
 public interface MultiStreamTypeEventListener<E, I> : EventListener {
-    public val streamTypes: List<StreamType<out E, I>>
+    public val streamTypes: List<StreamType<out E, out I>>
```

Comment:
`out I` here shouldn't be necessary, all id types in the same store in theory should be the same.

Reply:
You might be right, even if `I` is `Any` you need the `out`.